CRITICAL - 2024-10-28 04:12:54 --> ParseError: syntax error, unexpected identifier "view"
[Method: GET, Route: iniciodesesion]
in APPPATH\Controllers\Inicio.php on line 12.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 04:13:07 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "layout/header.php"
[Method: GET, Route: iniciodesesion]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('layout/header.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('layout/header', [], true)
 3 APPPATH\Controllers\Inicio.php(12): view('layout/header')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->iniciodesesion()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:06:37 --> ParseError: syntax error, unexpected token "if"
[Method: GET, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 41.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:06:45 --> ParseError: syntax error, unexpected token "echo"
[Method: GET, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 46.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:09:13 --> ParseError: syntax error, unexpected token "echo"
[Method: GET, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 46.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:09:33 --> ParseError: Unclosed '{' on line 8
[Method: GET, Route: registrodb]
in APPPATH\Models\ModeloMascota.php on line 37.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Models\\ModeloMascota.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\ModeloMascota')
 3 APPPATH\Controllers\Inicio.php(25): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\ModeloMascota')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:09:52 --> CodeIgniter\Exceptions\ModelException: "App\Models\ModeloMascota" model class does not have a valid dateFormat.
[Method: GET, Route: registrodb]
in SYSTEMPATH\BaseModel.php on line 1407.
 1 SYSTEMPATH\BaseModel.php(1407): CodeIgniter\Exceptions\ModelException::forNoDateFormat('App\\Models\\ModeloMascota')
 2 SYSTEMPATH\BaseModel.php(1381): CodeIgniter\BaseModel->intToDate(1730092192)
 3 SYSTEMPATH\BaseModel.php(830): CodeIgniter\BaseModel->setDate()
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-28 05:10:24 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ma...', 0)
#1 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ma...')
#2 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ma...')
#3 C:\xampp\htdocs\pfn\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ma...', Array, false)
#4 C:\xampp\htdocs\pfn\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pfn\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pfn\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pfn\app\Controllers\Inicio.php(43): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pfn\system\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
#9 C:\xampp\htdocs\pfn\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
#10 C:\xampp\htdocs\pfn\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pfn\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\pfn\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\pfn\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-10-28 05:10:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: GET, Route: registrodb]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:10:24 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:10:23\', \'2024-10-28 05:10:23\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:10:23\', \'2024-10-28 05:10:23\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:10:24 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:10:23\', \'2024-10-28 05:10:23\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:10:23\', \'2024-10-28 05:10:23\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:10:23\', \'2024-10-28 05:10:23\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-28 05:11:30 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ma...', 0)
#1 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ma...')
#2 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ma...')
#3 C:\xampp\htdocs\pfn\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ma...', Array, false)
#4 C:\xampp\htdocs\pfn\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pfn\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pfn\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pfn\app\Controllers\Inicio.php(43): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pfn\system\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
#9 C:\xampp\htdocs\pfn\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
#10 C:\xampp\htdocs\pfn\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pfn\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\pfn\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\pfn\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-10-28 05:11:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: GET, Route: registrodb]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:11:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:11:30\', \'2024-10-28 05:11:30\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:11:30\', \'2024-10-28 05:11:30\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:11:30 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:11:30\', \'2024-10-28 05:11:30\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:11:30\', \'2024-10-28 05:11:30\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:11:30\', \'2024-10-28 05:11:30\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-28 05:12:20 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ma...', 0)
#1 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ma...')
#2 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ma...')
#3 C:\xampp\htdocs\pfn\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ma...', Array, false)
#4 C:\xampp\htdocs\pfn\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pfn\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pfn\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pfn\app\Controllers\Inicio.php(43): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pfn\system\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
#9 C:\xampp\htdocs\pfn\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
#10 C:\xampp\htdocs\pfn\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pfn\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\pfn\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\pfn\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-10-28 05:12:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: GET, Route: registrodb]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:12:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:12:20\', \'2024-10-28 05:12:20\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:12:20\', \'2024-10-28 05:12:20\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:12:20 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:12:20\', \'2024-10-28 05:12:20\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:12:20\', \'2024-10-28 05:12:20\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:12:20\', \'2024-10-28 05:12:20\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-28 05:13:32 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ma...', 0)
#1 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ma...')
#2 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ma...')
#3 C:\xampp\htdocs\pfn\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ma...', Array, false)
#4 C:\xampp\htdocs\pfn\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pfn\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pfn\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pfn\app\Controllers\Inicio.php(43): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pfn\system\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
#9 C:\xampp\htdocs\pfn\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
#10 C:\xampp\htdocs\pfn\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pfn\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\pfn\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\pfn\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-10-28 05:13:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: GET, Route: registrodb]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:13:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:13:32\', \'2024-10-28 05:13:32\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:13:32\', \'2024-10-28 05:13:32\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:13:32 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:13:32\', \'2024-10-28 05:13:32\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:13:32\', \'2024-10-28 05:13:32\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:13:32\', \'2024-10-28 05:13:32\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `updated_at`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-28 05:13:41 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ma...', 0)
#1 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ma...')
#2 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ma...')
#3 C:\xampp\htdocs\pfn\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ma...', Array, false)
#4 C:\xampp\htdocs\pfn\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pfn\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pfn\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pfn\app\Controllers\Inicio.php(43): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pfn\system\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
#9 C:\xampp\htdocs\pfn\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
#10 C:\xampp\htdocs\pfn\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pfn\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\pfn\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\pfn\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-10-28 05:13:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'nombre' cannot be null
[Method: GET, Route: registrodb]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :fecha_actualiza:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:13:41 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:13:41\', \'2024-10-28 05:13:41\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:13:41\', \'2024-10-28 05:13:41\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :fecha_actualiza:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:13:41 --> [Caused by] mysqli_sql_exception: Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:13:41\', \'2024-10-28 05:13:41\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:13:41\', \'2024-10-28 05:13:41\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:13:41\', \'2024-10-28 05:13:41\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :fecha_actualiza:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-28 05:17:07 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\pfn\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ma...', 0)
#1 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ma...')
#2 C:\xampp\htdocs\pfn\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ma...')
#3 C:\xampp\htdocs\pfn\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ma...', Array, false)
#4 C:\xampp\htdocs\pfn\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pfn\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pfn\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pfn\app\Controllers\Inicio.php(43): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pfn\system\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
#9 C:\xampp\htdocs\pfn\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
#10 C:\xampp\htdocs\pfn\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pfn\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\pfn\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\pfn\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-10-28 05:17:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'nombre' cannot be null
[Method: GET, Route: registrodb]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :fecha_actualiza:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:17:07 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:17:07\', \'2024-10-28 05:17:07\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:17:07\', \'2024-10-28 05:17:07\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :fecha_actualiza:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:17:07 --> [Caused by] mysqli_sql_exception: Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:17:07\', \'2024-10-28 05:17:07\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:17:07\', \'2024-10-28 05:17:07\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, \'2024-10-28 05:17:07\', \'2024-10-28 05:17:07\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mascotas` (`nombre`, `especie`, `raza`, `fecha_nacimiento`, `sexo`, `color`, `peso`, `nombre_dueno`, `telefono_dueno`, `email_dueno`, `fecha_registro`, `fecha_actualiza`) VALUES (:nombre:, :especie:, :raza:, :fecha_nacimiento:, :sexo:, :color:, :peso:, :nombre_dueno:, :telefono_dueno:, :email_dueno:, :fecha_registro:, :fecha_actualiza:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Inicio.php(43): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:21:02 --> Error: Call to undefined method App\Controllers\Inicio::getPost()
[Method: GET, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 28.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:21:52 --> Error: Call to undefined method CodeIgniter\HTTP\IncomingRequest::get()
[Method: GET, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 28.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->registrobd()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:25:32 --> ParseError: syntax error, unexpected token ","
[Method: POST, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:25:51 --> ParseError: syntax error, unexpected token "="
[Method: POST, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:29:14 --> ParseError: syntax error, unexpected identifier "Registro", expecting "," or ";"
[Method: POST, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 45.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:29:36 --> ParseError: syntax error, unexpected identifier "Registro", expecting "," or ";"
[Method: POST, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 45.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:29:54 --> ParseError: syntax error, unexpected single-quoted string "iniciodesesion", expecting "," or ";"
[Method: POST, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 46.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:30:09 --> ParseError: syntax error, unexpected single-quoted string "iniciodesesion", expecting "," or ";"
[Method: POST, Route: registrodb]
in APPPATH\Controllers\Inicio.php on line 46.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:40:04 --> ParseError: syntax error, unexpected token "public"
[Method: GET, Route: index]
in APPPATH\Controllers\Inicio.php on line 18.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pfn\\app\\Controllers\\Inicio.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Inicio')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Inicio')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Inicio', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:41:18 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "dashboard.php"
[Method: GET, Route: sala]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('dashboard.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('dashboard', [], true)
 3 APPPATH\Controllers\Inicio.php(15): view('dashboard')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->sala()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:47:48 --> ParseError: syntax error, unexpected token "="
[Method: GET, Route: sala]
in APPPATH\Views\Administracion\dashboard.php on line 7.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Administracion/dashboard', [], true)
 3 APPPATH\Controllers\Inicio.php(15): view('Administracion/dashboard')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->sala()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:47:56 --> ErrorException: Undefined global variable $_SESSION
[Method: GET, Route: sala]
in APPPATH\Views\Administracion\dashboard.php on line 7.
 1 APPPATH\Views\Administracion\dashboard.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined global variable $_SESSION', 'C:\\xampp\\htdocs\\pfn\\app\\Views\\Administracion\\dashboard.php', 7)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\pfn\\app\\Views\\Administracion\\dashboard.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Administracion/dashboard', [], true)
 5 APPPATH\Controllers\Inicio.php(15): view('Administracion/dashboard')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->sala()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:48:26 --> ParseError: syntax error, unexpected token ";"
[Method: GET, Route: sala]
in APPPATH\Views\Administracion\dashboard.php on line 7.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Administracion/dashboard', [], true)
 3 APPPATH\Controllers\Inicio.php(15): view('Administracion/dashboard')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->sala()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:48:43 --> ParseError: syntax error, unexpected token ";"
[Method: GET, Route: sala]
in APPPATH\Views\Administracion\dashboard.php on line 7.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Administracion/dashboard', [], true)
 3 APPPATH\Controllers\Inicio.php(15): view('Administracion/dashboard')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Inicio->sala()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 05:57:31 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "index.php"
[Method: GET, Route: mascotas]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('index.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('index', [], true)
 3 APPPATH\Controllers\Mascotas.php(10): view('index')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Mascotas->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mascotas))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-28 06:09:01 --> ErrorException: Cannot redeclare App\Controllers\HistorialesMedicos::index()
[Method: GET, Route: historial]
in APPPATH\Controllers\HistorialesMedicos.php on line 13.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-10-28 06:09:53 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "index.php"
[Method: GET, Route: historial]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('index.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('index', [], true)
 3 APPPATH\Controllers\HistorialesMedicos.php(10): view('index')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\HistorialesMedicos->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HistorialesMedicos))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
